src/Exception/ChmodException.php

```diff
@@ -1,16 +1,16 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Webimpress\SafeWriter\Exception;
 
+use RuntimeException as PhpRuntimeException;
+
 use function sprintf;
 
-final class ChmodException extends \RuntimeException implements ExceptionInterface
+final class ChmodException extends PhpRuntimeException implements ExceptionInterface
 {
-    /**
-     * @param string $file
-     * @return self
-     */
-    public static function unableToChangeChmod($file)
+    public static function unableToChangeChmod(string $file) : self
```

Comment:
This could be made internal to avoid future BC breaks. You'll be able to remove or rename it at any time.

Reply:
Marked as internal and also changed all exceptions to be not instantiable (private construct).